frontend/assets/scss/style.scss

```diff
@@ -0,0 +1,80 @@
+// CSS reset largely copied from Tailwind CSS's Preflight: https://tailwindcss.com/docs/preflight
+
+body {
+    font-family: sans-serif;
+}
+
+body,
+blockquote,
+dl,
+dd,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
```

Comment:
nit: id personally put the h tags on the same line cause its easier to scan if these in particular are grouped
```css
body,
blockquote,
h1, h2, h3, h4, h5, h6,
dl,
dd,
...
```